nautobot/ipam/tests/test_models.py

```diff
@@ -112,33 +112,6 @@ def test_get_db_prep_value_mysql(self):
         self.assertEqual(prepped, manual)
 
 
-class TestAggregate(TestCase):
-    def test_get_utilization(self):
```

Comment:
Should any of this test content port over to TestPrefix.test_get_utilization?

Reply:
Most of this logic is already covered there but we could add one of the larger networks to make sure we don't overflow ints?